crates/bevy_window/src/window.rs

```diff
@@ -803,22 +836,24 @@ pub enum CompositeAlphaMode {
 )]
 #[reflect(Debug, PartialEq)]
 pub enum WindowMode {
-    /// Creates a window that uses the given size.
+    /// The window should take a portion of the screen, using the window resolution size.
     #[default]
     Windowed,
-    /// Creates a borderless window that uses the full size of the screen.
+    /// The window should be borderless and use the full size of the screen.
     BorderlessFullscreen,
-    /// Creates a fullscreen window that will render at desktop resolution. The app will use the closest supported size
-    /// from the given size and scale it to fit the screen.
+    /// The window should be borderless, take the full size of the screen, and render at desktop resolution.
+    /// The app will use the closest supported size from the window resolution size and scale it to fit the screen.
```

Comment:
I believe it's more something like:

```suggestion
    /// The app will use the closest screen resolution supported by the monitor, and will set the window size and scale to match it.
```

No?

Reply:
See conversation here: https://github.com/bevyengine/bevy/pull/8858#discussion_r1232413155
I think all of the fullscreen variants documentation should be clarified not just this one but yeah right now they're not great ><'